migrations/20210912131923_init/migration.sql

```diff
@@ -137,7 +137,7 @@ ALTER TABLE "beneficiaries" ADD CONSTRAINT "beneficiaries_personId_fkey" FOREIGN
 ALTER TABLE "beneficiaries" ADD CONSTRAINT "beneficiaries_coordinatorId_fkey" FOREIGN KEY ("coordinatorId") REFERENCES "coordinators"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
 
 -- AddForeignKey
-ALTER TABLE "campaigns" ADD CONSTRAINT "campaigns_beneficiary_id_fkey" FOREIGN KEY ("beneficiary_id") REFERENCES "beneficiaries"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
```

Comment:
Please restore the initial version of the migration as it was before. If we need to change something in the database it needs to happen in the following way:

1. Update of `schema.prisma
2. Generate client and entities `yarn generate-schema`
3. Generate new migration with `yarn prisma migrate dev`

Reply:
Done.